k8s/namespaces/nfdiv/nfdiv-frontend/demo.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-frontend
+spec:
+  releaseName: nfdiv-frontend
+  chart:
```

Comment:
oops, also should remove this, saw just as i clicked approve

Reply:
will remove that :)